### ISSUE TITLE
Refactor rand_exercise endpoint to send query params

### ISSUE THREAD

**Issue by sienna-kopf** [SOM-Timer/som_timer_be#40]
### Feature Categories:
- [x] Endpoint

**Describe the feature/Bug your gonna work on**
- Refactor the /rand_exercise endpoint to send duration and category as query params. 
  - makes a little more sense for a get request 
  - includes a little bit of verbose error handling
  - also works to send in body under this structure which is great!

![Screen Shot 2020-10-29 at 11 29 28 AM](https://user-images.githubusercontent.com/62857073/97610094-0f963800-19da-11eb-9594-7f6d5b1d9b92.png)

**Additional context**
Add any other context or screenshots about the feature.
tldr; @Chulstro is still a genius
